Lesson2/LessonNotes

```diff
@@ -0,0 +1,3 @@
+https://docs.python.org/3/library/stdtypes.html#str
```

Comment:
In addition please do not mix pull request with task and code that you would like just to add to your repo.
Add code you able directly to main branch without any pull request.
PR ( pull request) I have asked just for hometasks checking. So better do not mix these things.

As advanced level try to play to modify current PR to have only homestask file without closing pull request.
So you add changes /fixes and just ping me again for re-review.